examples/librilight/README.md

```diff
@@ -0,0 +1,114 @@
+## Download data
+
+### Audio
+
+You can download librilight audios here : https://github.com/facebookresearch/libri-light/tree/main/data_preparation#1a-downloading
+
+It has three subsets, small (577 hours), medium (5193 hours), large (51934 hours).
+
+
+### Texts
+
+@xiaoyu
```

Comment:
@marcoyang1998 Could you write some notes (maybe upload the scripts) about how to download and clean the texts.

Reply:
I will upload them as soon as I finish cleaning the code.